test/protobuf/protoc/proto_gen/extension.pb.ex

```diff
@@ -22,6 +23,7 @@ defmodule Protobuf.Protoc.ExtTest.Dual do
           a: String.t() | nil,
           b: Google.Protobuf.StringValue.t() | nil
         }
+
```

Comment:
nit: lets try to minimize changes, maybe revert this formatting

Reply:
reverted!